modules/ROOT/pages/live-booting-ipxe.adoc

```diff
@@ -0,0 +1,57 @@
+= Booting live Fedora CoreOS via iPXE
+
+This guide shows how to boot a transient Fedora CoreOS (FCOS) system via iPXE. By default this will run FCOS in a stateless way, completely out of RAM. Separately, FCOS can also be installed to disk.
+
+== Prerequisite
+
+Before booting FCOS, you must have an Ignition configuration file and host it somewhere (e.g. on a reachable web server). If you do not have one, see xref:producing-ign.adoc[Producing an Ignition File].
+
+NOTE: The live system requires at least 3 GiB of RAM.
+
+== PXE images
+
+NOTE: Before August 2020, the Fedora CoreOS PXE image included two components: a `kernel` image and an `initramfs` image.  Beginning August 11, a third `rootfs` image will be added.  During an initial migration period, the `rootfs` image will be optional, and Fedora CoreOS will display a warning on login if the system was booted without it.  After the migration period, the `rootfs` image will be mandatory and the live PXE system will not boot without it.
+
+The migration timeline is:
+
+- August 11: `rootfs` image available in `next`, `testing`, and `stable` streams
+- August 25: `rootfs` image required in `next` stream
+- September 22: `rootfs` image required in `testing` stream
+- October 6: `rootfs` image required in `stable` stream
+
+To boot with the `rootfs` artifact, make one of the following changes to your PXE config:
+
+- Specify both `initramfs` and `rootfs` files as initrds in your PXE configuration.  In PXELINUX, put both file paths in the `initrd` directive, separated by commas, or specify two `initrd=` parameters in your `append` line.
+- Concatenate the `initramfs` and `rootfs` files together, and specify the combined file as the initrd.
+- Specify only the `initramfs` file as the initrd, and pass an HTTP(S) URL for the `rootfs` using the `coreos.live.rootfs_url=` kernel argument.
+
+For more information on this change, see the https://github.com/coreos/fedora-coreos-tracker/issues/390[tracker issue].
```

Comment:
This looks similar to the content from https://github.com/coreos/fedora-coreos-docs/pull/112. Should we just have it in one place and reference it?

Reply:
Yes, it is exactly the same content, I've been a bit lazy.